### DISCUSSION TITLE
Respect MacOS System Appearance Theme

### DISCUSSION THREAD

**Discussion by ngurney** [vercel/hyper#6082] [Q&A] [answered]
I regularly switch my Mac between dark and light modes. Most apps respect this and allow me to specify a theme for each. 

I'd love to see Hyper have an option for a Light theme, and a Dark theme, that would switch based on my OS setting. 

Is there a plugin that does this? Or perhaps something that Hyper would consider building natively?

**Comment by martindzejky** [comment, marked as answer]
Maybe https://github.com/tbremer/hyper-prefers-color-scheme?
